README.md

```diff
@@ -27,6 +27,8 @@ compatibility list:
 
 | wasmd | wasmvm       | cosmwasm-vm | cosmwasm-std |
 |-------|--------------|-------------|--------------|
+| 0.44  | v1.5.0       |             | 1.0-1.5      |
```

Comment:
Oh, so we release it in 0.44 already, not 0.50?

Reply:
we are not yet ready for 0.50 . Since we want to release it very soon, it needs to go to v0.44